src/main/resources/templates/header.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Header</title>
+    <meta charset="UTF-8">
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css'>
```

Comment:
Don't you already bring Bootstrap through the fragment you're referencing a few lines below?

Reply:
Removed duplicates links